app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,412 @@
+  font-weight: 525;
+  line-height: 20px;
+  border: 0;
+}
+#menu-top li:hover {
+  transform: scale(1.2);
+}
+.navbar-toggler {
+  filter: invert(100%);
+  box-shadow: 0px 0px 10px black;
+  transition: transform .2s;
+}
+.navbar-toggler:hover {
+  transform: scale(1.1);
+}
+.fa-utensils {
+  color: white;
+}
+@media (max-width: 992px) {
+  .bg-custom {
+    background-image: url('bg-dropdown.jpg');
+    background-size: cover;
+    background-position: center center;
+    border-radius: 0 0 15px 15px;
+    box-shadow: 0 0 10px white;
+  }
+  .middot {
+    visibility: hidden;
+  }
+}
+.middot {
+  color: orange;
+  border: 0;
+}
+.score {
+  display: inline-flex;
+  flex-wrap: wrap;
+  width: fit-content;
+  padding: 5px;
+  font-size: 13px;
+  align-content: center;
+}
+.service-header {
+  background-image: url('coffee-shop.jpg');
+  background-size: cover;
+  background-position: center center;
+  width: 100%;
+}
+/* Navbar container */
+
+.sorting-nav {
+  /*overflow: visible;*/
+  background-color: #333;
+}
+/* Links inside the navbar */
+
+.sorting-nav a {
+  float: left;
+  font-size: 16px;
+  color: black;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+/* The dropdown container */
+
+.dropdown {
+  border-radius: 5px;
+  float: left;
+  overflow: visible;
+  padding-right: 20px;
+  padding-left: 20px;
+}
+
+/* Dropdown button */
+
+.dropdown .dropbtn {
+  letter-spacing: 2px;
+  font-size: 9px;
+  font-weight: bold;
+  text-transform: uppercase;
+  font-family: Verdana !important;
+  border: none;
+  outline: none;
+  color: black;
+  padding: 5px 5px;
+  background-color: inherit;
+  font-family: inherit;
+  /* Important for vertical align on mobile phones */
+  margin: 0;
+  /* Important for vertical align on mobile phones */
+}
+.sorting-nav a:hover, .dropdown:hover .dropbtn {
+  background-color: #c2c3c4;
+  border-radius: 5px;
+}
+.sorting-nav a, .dropdown .dropbtn {
+  background-color: none;
+}
+/* Dropdown content (hidden by default) */
+
+.dropdown-content {
+  display: none;
+  position: absolute;
+  background-color: #f9f9f9;
+  /*  min-width: 160px;
+*/
+  box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+  z-index: 1;
+}
+/* Links inside the dropdown */
+
+.dropdown-content a {
+  float: none;
+  color: black;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: block;
+  text-align: left;
+}
+/* Add a grey background color to dropdown links on hover */
+
+.dropdown-content a:hover {
+  background-color: #cccccc;
+}
+/* Show the dropdown menu on hover */
+
+.dropdown:hover .dropdown-content {
+  display: block;
+}
+.fa-angle-down {
+  color: orange;
+}
+
+div.slidecontainer {
+  height: 10px;
+}
+div.range-slider {
+  position: relative;
+  width: 200px;
+  height: 10px;
+  float: left;
+  text-align: center;
+}
+div.range-slider input[type="range"] {
+  pointer-events: none;
+  position: absolute;
+  -webkit-appearance: none;
+  -webkit-tap-highlight-color: rgba(255, 255, 255, 0);
```

Comment:
is this code you wrote?

Reply:
this is from a code snippet used for the range slider with two thumbs. 
it was made as two different sliders which overlap where the two thumbs have a higher z-index in order to come upfront and make it look like a singer slider with two thumbs. 
i'll dispose of it if needed. ^_^